lib/src/services/jira.ts

```diff
@@ -146,7 +183,212 @@ export const Issue = T.type({
   ),
 });
 
-export type Issue = T.TypeOf<typeof Issue>;
+export const CloudJiraIssue = T.type({
```

Comment:
"Generic" types should have no prefix (those will be the commonly used types in the rest of the system).

Reply:
Does this mean change the wording of `GenericJiraIssue` back to `Issue`?

Reply:
changed from `GenericJiraIssue` to `JiraIssue`